src/net/sourceforge/kolmafia/request/PlaceRequest.java

```diff
@@ -339,6 +342,34 @@ else if (responseText.contains("a giant jagged hole in the wall")) {
     SpadingManager.processPlace(urlString, responseText);
   }
 
+  private static void parseSotVisit(String responseText) {
+    String part1 = "What he means is that he lost something over in ";
```

Comment:
Can you not just use a regex here? It's much simpler and we do it everywhere

Reply:
We can.  I can't.  I had concerns about whether this code was even in the right place and whether the test structure did things in the current preferred way.  I figured I'd get that out of the way first.  Since it looks like this is going to "survive" I'll force myself to relearn enough regex to redo this.